ahnlich/grpc_types/build.rs

```diff
@@ -0,0 +1,123 @@
+use std::{
+    collections::HashSet,
+    io::{Result, Write},
+    path::PathBuf,
+};
+use walkdir::WalkDir;
+
```

Comment:
Add a `TODO` or `FIXME` here that shows how we aim to replace `types` with `grpc_types` i.e turn the latter into the former